taigaio/taiga-back
# feat(csv userstory): add swimlane to the csv output

![](https://media.giphy.com/media/hbrCfxHxOhiZW/giphy.gif)

This PR just add swimlane in the userstories csv response.

### How to validate:

- [x] Run a local taiga instance.
- [x] Get the Userstories csv file: 
  - Access to a projects with swimlanes and user stories (for example project 1).
  - Go to Admin > Project > Report
  - Generate the user stories report url and download it
  - Check the Swimlane colunm exists
- [x] Move [Task #951](https://tree.taiga.io/project/taiganext/task/951?kanban-status=1857763) to ready for qa.
